### PR TITLE
[tools] add reboot playbook to reboot DebOps hosts

### PR DESCRIPTION
**Pull request by nqb**
As discussed on #1475

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -42,6 +42,13 @@ New DebOps roles
 
   .. __: https://www.elastic.co/beats/filebeat
 
+General
+'''''''
+
+- The :file:`tools/reboot.yml` can be used to reboot DebOps hosts even if they
+  are secured by ``molly-guard`` package.
```
Comment (tasosalvas) [inline CHANGELOG.rst]: the

**Review by tasosalvas — DISMISSED** [review]
Just that little typo. I'm going to bed :)

File: CHANGELOG.rst (hunk 1 above)
Comment (tasosalvas) [reply CHANGELOG.rst]: Sorry, I was unclear, I meant "by the ``molly-guard`` package."

**Review by tasosalvas — DISMISSED** [review]

**Comment by tasosalvas** [comment]
Hey I just realized we have a doc for playbooks and `tools/` are undocumented.

Would you like to add a "tools" section at the bottom of `docs/user-guide/playbooks.rst` and have a go at documenting what's currently in there? Like, a paragraph that those are meant to be ran manually then a few words about each, so end users can know what they do without reading the source.

It's ok if that's too much, just thought I'd drop the idea. :) I will get around to it eventually otherwise.

**Review by drybjed — APPROVED** [review]
Sounds good, thanks. The `reboot` task didn't want to complete properly, so I'll try to fix it in a patch.

**Pull request merged**